arrow-libs/fx/arrow-fx-dsl/src/commonMain/kotlin/arrow/scoped/ScopingScope.kt

```diff
@@ -0,0 +1,56 @@
+// Import throwIfFatal from AutoCloseScope
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
+
+package arrow.scoped
+
+import arrow.AutoCloseScope
+import arrow.throwIfFatal
+import arrow.atomic.Atomic
+import arrow.atomic.update
+import kotlinx.coroutines.NonCancellable
+import kotlinx.coroutines.withContext
+import kotlin.coroutines.cancellation.CancellationException
+
+public interface ScopingScope : AutoCloseScope {
```

Comment:
This is identical to `ResourceScope`, no? `ScopingScope` doesn't feel like a better name IMO.